internal/task/task/common/client_status.go

```diff
@@ -100,18 +174,25 @@ func NewGetClientStatusTask(curveadm *cli.CurveAdm, v interface{}) (*task.Task,
 
 	// add step
 	var status string
+	var port string
 	t.AddStep(&step.ListContainers{
 		ShowAll:     true,
 		Format:      `"{{.Status}}"`,
 		Filter:      fmt.Sprintf("id=%s", containerId),
 		Out:         &status,
 		ExecOptions: curveadm.ExecOptions(),
 	})
+	t.AddStep(&step2Port{
```

Comment:
Maybe `step2GetPort` is more meaningful.

Reply:
done